backend/clubs/management/commands/import_calendar_events.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand
+
+from clubs.models import Club
+
+
+class Command(BaseCommand):
```

Comment:
@rohan-gupta8 Add a `help` attribute to this command, see any of the other commands for an example. Should be a description of what this script does, will show up when you do `./manage.py import_calendar_events --help`.